src/bench_databuf.rs

```diff
@@ -0,0 +1,32 @@
+use criterion::{black_box, Criterion};
+use databuf::{config::num::LE, *};
+
+pub fn bench<T>(name: &'static str, c: &mut Criterion, data: &T)
+where
+    T: Encode + for<'de> Decode<'de>,
+{
+    const BUFFER_LEN: usize = 10_000_000;
+
+    let mut group = c.benchmark_group(format!("{}/databuf", name));
+
+    let mut serialize_buffer = Vec::with_capacity(BUFFER_LEN);
+    group.bench_function("serialize", |b| {
+        b.iter(|| {
+            serialize_buffer.clear();
+            serialize_buffer = data.to_bytes::<LE>();
+            black_box(serialize_buffer.clone())
```

Comment:
```suggestion
            black_box(data.encode::<LE>(&mut serialize_buffer));
```

Cloning the serialize buffer is very expensive and will skew the results. This will guarantee that the space allocated for the serialize buffer gets reused.